.github/workflows/CI.yml

```diff
@@ -91,6 +91,7 @@ jobs:
           name: coverage-data-${{ matrix.python-version }}
           path: .coverage
           if-no-files-found: ignore
+          include-hidden-files: true
```

Comment:
`actions/upload-artifact` recently added this key which broke the upload for `.coverage`

Reply:
curious if they said why they default to `false`, thereby breaking backward compatibility?